README.rst

```diff
@@ -53,7 +53,7 @@ To install ARIA from source, download the source tarball from
 **Ubuntu/Debian** (tested on Ubuntu14.04, Ubuntu16.04)::
 
     apt-get install -y python-dev gcc libffi-dev libssl-dev
-    pip install aria[ssh]
+    pip install apache-ariatosca[ssh]
```

Comment:
This change should also be applied to the other platforms - i.e. it should be "apache-ariatosca" on centos/windows/.. as well.